packages/itwinui-react/CHANGELOG.md

```diff
@@ -42,9 +42,7 @@
 
 - 8e319bea: Removes transitions for `Toast` component when `prefer-reduced-motion` is active
 - 24a0cf94: Fixed an issue where Table's `onBottomReached` callback was not being invoked when using strict mode with react 18.
-- Updated dependencies [f7caf384]
-- Updated dependencies [dd13257f]
-- Updated dependencies [ec26b72d]
```

Comment:
I don't know why it's adding multiple entries for updating dependencies. Will need to investigate in the future.